### DISCUSSION TITLE
Anchor tags added/repeated in rendered markup

### DISCUSSION THREAD

**Discussion by hyphen81** [livewire/livewire#6191] [1. Help]
I have the below blade file that has a foreach loop. I've noticed that livewire is repeating the anchor tag within the markup and I don't know why. I've using `wire:key` and it's unique to the page. I even added `wire:key` to the anchor itself but it's still doing it. Has anyone else ever seen this, and if so, how do I fix it? I'm totally lost.

Here's my markup:

```
<div class="row">
		@foreach ($templates as $template)
			<div class="col-sm-12 col-md-6 col-xl-3 mb-4" wire:key="template-id-{{ $template->id }}">
				<a href="{{ route('locations.request', ['location' => $location, 'request' => $template]) }}" wire:key="template-link-{{ $template->id }}">
					<div class="card">
						<div class="card-header">
							<h5 class="text-uppercase">{{ $template->request_name }}</h5>
						</div>
						<div class="card-body">
							<h6><i class="mdi mdi-checkbox-marked-circle h5 text-muted me-2"></i>0</h6>
							@if ($template->frequency_type_id == 2)
								<h6><i class="mdi mdi-calendar-clock h5 text-muted me-2"></i>Runs weekly on {{ jddayofweek($template->start_day_of_wek, 1) }}s</h6>
							@else
								<h6><i class="mdi mdi-calendar-clock h5 text-muted me-2"></i>Monthly on the {{ $template->ordinal_frequency }} day</h6>
							@endif

							<h6><i class="mdi mdi-calendar-check h5 text-muted me-2"></i>Next run on {{ Helper::dateForHumans($template->next_run_date) }}</h6>
						</div>
						<div class="card-footer">
							<div class="d-flex justify-content-between align-items-center">
								<a role="button" id="delete-btn-{{ $template->id }}" class="delete-btn btn btn-outline-danger btn-xs me-2" data-bs-toggle="modal" data-bs-target="#confirmDelete">
									Delete
								</a>

								<a role="button" id="enable-btn-{{ $template->id }}" class="icon-button {{ $template->is_enabled ? 'text-success' : 'text-danger' }} me-2" wire:click="toggleEnabled({{ $template->id }})" data-tippy-content="{{ $template->is_enabled ? 'Enabled' : 'Disabled' }}">
									<i class="mdi mdi-power h3"></i>
								</a>
							</div>
						</div>
					</div>
				</a>
			</div>
		@endforeach
	</div>
```

And here's what comes out as the rendered html from dev tools (the green outline is the one that should be there, all the others are a mystery to me:

![repeated anchor tags](https://github.com/livewire/livewire/assets/10478061/c189dd13-84ef-4a0f-a916-4e6d31f5425b)

If I comment out the anchor tag, all the "phantom" tags go away, so I know it's related to the anchor tag and not some other aspect of the markup. I've also tried removing everything in the anchor tag so it's literally just `<a>` or `<a href="#">` and the result is the same. The extra tags don't go away unless I remove the main anchor tag entirely.

**Comment by hyphen81** [comment]
I ended up just taking the link off of the card itself and having a link in the footer to the route. Frustrating, but livewire(?) doesn't seem to like the fact that I had an anchor wrapping the card and then had anchors in the footer as well. Wasted way too much time on this.

**Comment by citricguy** [comment]
Nesting anchor tags like will cause you problems down the road. I learned the hard way due to how apple handles this in some cases on iOS devices. 😢 Same goes for `<button>`.
